### PR TITLE
Eithar- misscat and tests

### PR DESCRIPTION
**Pull request by EitharAlfatih**

### CODE REVIEW

File: R/Untitled.R [hunk 1]
```diff
@@ -0,0 +1,3 @@
+fbind <- function(a, b) {
+        factor(c(as.character(a), as.character(b)))
+}
```
Comment (doraqmon) [inline R/Untitled.R]: This file is not needed in our repo.

Comment (doraqmon) [reply R/Untitled.R]: Could you remove it ?

File: R/misscat.R [hunk 2]
```diff
@@ -10,5 +10,23 @@
...
+
+        rows <- apply(df, 1, function(x) mean(is.na(x))) < threshold
+        df <- df[rows, ]
+        return(df)
+
 }
+
+
+
+
```
Comment (doraqmon) [inline R/misscat.R]:
```suggestion
```
remove extra lines

**Review by doraqmon — CHANGES_REQUESTED** [review]
The functions look good. Just need to remove your working untitle.R file...

File: R/Untitled.R (hunk 1 above)
Comment (EitharAlfatih) [reply R/Untitled.R]: Done

**Review by doraqmon — APPROVED** [review]

**Pull request merged**